src/launchpad/size/analyzers/apple.py

```diff
@@ -140,6 +145,9 @@ def analyze(self, artifact: AppleArtifact) -> AppleAnalysisResults:
             )
             insights = AppleInsightResults(
                 duplicate_files=DuplicateFilesInsight().generate(insights_input),
+                large_audio=LargeAudioFileInsight().generate(insights_input),
```

Comment:
This wasn’t added to the apple analysis in the other PRs so I added all of them.